### PR TITLE
add Alias tooltip on AS result

### PR DESCRIPTION
**Pull request by anistouri**

### CODE REVIEW

File: src/components/security-analysis-result.js [hunk 1]
```diff
@@ -211,11 +216,31 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
...
+            params.data?.contingencyEquipmentsIds
+        ) {
+            const values = Object.values(params.data?.contingencyEquipmentsIds);
+            const valuesToDisplay = values?.join(', ');
+
+            return {
+                title: null,
+                values: valuesToDisplay,
+            };
+        } else {
```
Comment (AAJELLAL) [inline src/components/security-analysis-result.js:232]: You can retun null without the "else"

File: src/components/security-analysis-result.js [hunk 2]
```diff
@@ -211,11 +216,31 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
         },
         [onClickNmKConstraint, classes.button]
     );
+
+    const toolTipValueGetterValues = (params) => {
+        if (
+            params?.data?.contingencyId &&
```
Comment (AAJELLAL) [inline src/components/security-analysis-result.js:222]: Do you really need all the optional chaining?
If yes, you need to add "?" for : params.data?.contingencyEquipmentsIds

File: src/components/custom-aggrid/custom-tooltip-values.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { Grid, Typography, Tooltip } from '@mui/material';
+import { ITooltipParams } from 'ag-grid-community';
+import { FunctionComponent } from 'react';
+
+const CustomTooltipValues: FunctionComponent<ITooltipParams> = (props) => {
+    const values: string[] = props?.value?.values;
+    return (
+        <>
+            {values && values.length && (
```
Comment (AAJELLAL) [inline src/components/custom-aggrid/custom-tooltip-values.tsx:9]: I think it'd better to use values?.length

**Review by dbraquart — APPROVED** [review]
code: ok

tests:
	- with explicit naming list: ok
	- with criteria-based list: it works too, but I think the PO does not want a tooltip in this case, because the element Id is always like the contengency Id. The thing is we dont know in which case we are when we look at the SA result => to be discussed with the PO.

File: src/components/security-analysis-result.js (hunk 1 above)
Comment (anistouri) [reply src/components/security-analysis-result.js]: Done

File: src/components/custom-aggrid/custom-tooltip-values.tsx (hunk 3 above)
Comment (anistouri) [reply src/components/custom-aggrid/custom-tooltip-values.tsx]: Done

**Review by AAJELLAL — APPROVED** [review]

**Pull request merged**